CloudSQLReplicator/google/cloud/pontem/sql/replicator/util/gcp_api_util.py

```diff
@@ -43,6 +46,8 @@
 DEFAULT_1ST_GEN_REGION = 'us-central'
 DEFAULT_2ND_GEN_REGION = 'us-central1'
 
+#REGEX
```

Comment:
"REGEX" as a comment is not useful since I can see that the next line is a Regex. I would provide more context, such as what is the RFC? what is it for, etc.?